### PR TITLE
Feature/add localization

### PR DESCRIPTION
**Pull request by LostFan**

### CODE REVIEW

File: src/main/webapp/js/app.js [hunk 1]
```diff
@@ -118,7 +120,100 @@ angular
             parent: 'root'
         });
 
-        }
+        $translateProvider.translations('en', {
```
Comment (Hanna-Aliakseichykava) [inline src/main/webapp/js/app.js]: May be, it makes sense to move translations in separate files - 1 file for 1 language.

Comment (LostFan) [inline src/main/webapp/js/app.js]: Done

**Comment by kmih** [comment]
:+1:

**Comment by Hanna-Aliakseichykava** [comment]
:+1:

**Pull request merged**